ext/dapr-ext-workflow/dapr/ext/workflow/dapr_workflow_client.py

```diff
@@ -37,9 +38,14 @@ class DaprWorkflowClient:
        This client is intended to be used by workflow application, not by general purpose
        application.
     """
-    def __init__(self, host: Optional[str] = None, port: Optional[str] = None):
+    def __init__(self, host: Optional[str] = None, port: Optional[str] = None, apiToken: Optional[str] = None):
         address = getAddress(host, port)
-        self.__obj = client.TaskHubGrpcClient(host_address=address)
+        metadata = []
```

Comment:
```suggestion
        metadata = tuple()
```

This syntax means list, what you mean is a tuple, or else this will not pass type checking.

Reply:
thanks for the suggestion, changed.